conductor/lib/maya_utils.py

```diff
@@ -516,42 +526,55 @@ def collect_dependencies(node_attrs):
                     # directory (i.e. it doesn't have any real smarts about path resolution, etc).
                     # NOTE: that this command will oftentimes return filepaths with an ending "/" on
                     # it for some reason. Strip this out at the end of the function
-                    path = cmds.file(plug_value, expandName=True, query=True, withoutCopyNumber=True)
+                    path = os.path.normpath(cmds.file(plug_value, expandName=True, query=True, withoutCopyNumber=True))
                     logger.debug("%s: %s", plug_name, path)
-
+                    
+                    if path in exclude_path_list:
+                        logger.info("Skipping depedency '{}' - in exclusion list".format(path))
+                        continue
+                    
+                    if path in leaf_path_list:
+                        logger.info("Skipping nested scanning of '{}' - in leaf list".format(path))
```

Comment:
The log entry message doesn't match the code.

Reply:
If you `continued`'d here, you'd be able to remove the duplicated `if path not in leaf_path_list:` in this function.

Reply:
No.. because the path itself still needs to be added to the list of dependencies on line 640. I struggled on how to do this cleanly... It really needs a class-based approach. I know @hoolymama did a big refactor on the dependency scanner. Perhaps we can merge that in to take advantage of the refactor?